pages/common/gradle.md

```diff
@@ -1,15 +1,27 @@
 # gradle
 
-> Gradle is the official build system for Android Studio.
+> Gradle is an advanced open source build automation system.
 
 - Compile a package:
 
 `gradle build`
 
+- Exclude test task:
+
+`gradle build -x test`
```

Comment:
Shouldn't `test` be inside tokens (`{{}}`) ?

Reply:
Thanks. Done.